### PR TITLE
Feature/zenodo workflow fixes

### PR DESCRIPTION
**Pull request by davidorme**
This PR is mostly tuning `safedata_validator` to work cleanly with the metadata webserver implementation. There are some documentation updates, some minor tweaks and a rewrite of the `safedata_post_metadata` entry point to add functionality to update the gazetteer data stored on the webserver.

### CODE REVIEW

File: docs/command_line_tools/command_line_usage/safedata_server_top.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cl_prompt $ safedata_server -h
+usage: safedata_server [-h] [-r RESOURCES] [-q]
+                       {post_metadata,update_gazetteer} ...
+
+Post updated information to a safedata server instance/
```
Comment (jacobcook1995) [inline docs/command_line_tools/command_line_usage/safedata_server_top.txt:5]: Should this be a slash at the end?

File: mkdocs.yml [hunk 2]
```diff
@@ -27,9 +27,10 @@ nav:
       - 'Other file formats': 'data_providers/data_format/other_formats.md'
       - 'Data format FAQs': 'data_providers/data_format/faqs.md'
   - 'Command line tools':
-    - 'overview': 'command_line_tools/overview.md'
+    - 'Overview': 'command_line_tools/overview.md'
     - 'safedata_validate': 'command_line_tools/safedata_validate.md'
     - 'safedata_zenodo': 'command_line_tools/safedata_zenodo.md'
+    - 'safedata_server': 'command_line_tools/safedata_server.md'
```
Comment (jacobcook1995) [inline mkdocs.yml:33]: Has this markdown file been added yet, I can't see it in the list of files changed

**Review by jacobcook1995 — COMMENTED** [review]
LGTM. My comments are more queries than requested changes so feel free to ignore them. As a broader query, not that much seems to have changed in the package code, am I right in thinking that is because most of the changes are on the metadata server end and the changes here are mainly to allow `sdv` to leverage those changes?

**Comment by jacobcook1995** [comment]
Oh and also does this pull request fix #36? (and also #25?)

File: docs/command_line_tools/command_line_usage/safedata_server_top.txt (hunk 1 above)
Comment (davidorme) [reply docs/command_line_tools/command_line_usage/safedata_server_top.txt]: Huh - fixed a flake8 typo and forgot to update usage

**Comment by davidorme** [comment]
No - doesn't fix either #36  or #25.

**Comment by jacobcook1995** [comment]
> No - doesn't fix either #36 or #25.

Ahh right (I must admit to not fully grasping #25). Does this not fix #36 because the `update_gazetteer` function defined in `zenodo.py` wouldn't get a proper response from the server even if it was run with valid data?

**Comment by davidorme** [comment]
> Does this not fix #36 because the `update_gazetteer` function defined in `zenodo.py` wouldn't get a proper response from the server even if it was run with valid data?

No - the issue is that `sdv` used to get this data from the server (when _that_ was the source of truth) and used a simplified version of the full GeoJSON. But now the server is just a metadata server, admins should be maintaining that GeoJSON resource and using it directly with `sdv` and pushing updates to the server as required.

**Review by jacobcook1995 — APPROVED** [review]

**Pull request merged**